app/v3/controllers/validators/SubmitForeignPropertyBsasRulesValidator.scala

```diff
@@ -64,7 +64,7 @@ object SubmitForeignPropertyBsasRulesValidator extends RulesValidator[SubmitFore
   private val resolveAdjustment = ResolveParsedNumber(min = -99999999999.99, disallowZero = true)
 
   private def resolveAdjusted(path: String, value: Option[BigDecimal]): Validated[Seq[MtdError], Unit] =
-    resolveAdjustment(value, path = Some(path)).map(_ => ())
+    resolveAdjustment(value, path).map(_ => ())
```

Comment:
can use ".toUnit" from RulesValidator -- though now I'm wondering if much (or all) of RulesValidator could instead be moved into ResolverOps int ResolverSupport?  including toUnit etc

Reply:
Possibly. Though I didn't touch it because it felt out of scope. But actually it looks like many of these `.map(_ => ())` can simply be removed if they're just being fed into `combine` as it wildcards the right-hand type argument of `Validated`.